apps/pipelines/nodes/nodes.py

```diff
@@ -58,7 +58,21 @@
 OptionalInt = Annotated[int | None, BeforeValidator(lambda x: None if isinstance(x, str) and len(x) == 0 else x)]
 
 
-class RenderTemplate(PipelineNode):
+class OutputMessageTagMixin(BaseModel):
+    tag: str = Field(
```

Comment:
this value should receive the same unicode normalization treatement as in the tag views:

`unicodedata.normalize("NFC", value)`

You can do this with an `after` validator (https://docs.pydantic.dev/dev/concepts/validators/#field-validators)